app/src/main/java/com/framgia/soundclound/screen/home/HomeViewModel.java

```diff
@@ -1,21 +1,27 @@
 package com.framgia.soundclound.screen.home;
 
+import android.content.Context;
+import android.content.Intent;
```

Comment:
ERROR: Unused import: android.content.Intent. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports